File: test_javaPrj_01/src/programmers/lv0/배열원소의길이.py
    # @SuppressWarnings("unused")
    # public void Solution() throws Exception{

    #     String[] strlist = {"We", "are", "the", "world!"};

    #     int[] list = Arrays.stream(strlist).mapToInt(str -> str.length()).toArray();

    #     System.out.println(Arrays.toString(list));

    # }


def 배열원소의길이():
    
    strlist = ["We", "are", "the", "world!"]

    newArr = []
    for element in strlist:
        print(element.count)
        print(len(element))
        newArr.append(len(element))

    print(newArr)

    # 다른사람 아이디어 1
    # return [len(str) for str in strlist]

    # 다른사람 아이디어 2
    # return list(map(len, strlist))

    pass


배열원소의길이();

File: test_javaPrj_01/src/programmers/lv0/배열원소의길이.java
package programmers.lv0;

import java.util.Arrays;

public class 배열원소의길이 {
    
    private 배열원소의길이(){
        try{
            Solution();
            
        }catch(Exception e){
            e.printStackTrace();
        }
    }

    @SuppressWarnings("unused")
    public void Solution() throws Exception{

        String[] strlist = {"We", "are", "the", "world!"};

        int[] list = Arrays.stream(strlist).mapToInt(str -> str.length()).toArray();

        System.out.println(Arrays.toString(list));

    }


    public static void main(String[] args) {
        new 배열원소의길이();
    }


}
